src/nnet2bin/nnet1-to-raw-nnet.cc

```diff
@@ -28,6 +28,7 @@
 #include "nnet2/nnet-component.h"
 
 namespace kaldi {
+  bool affinecompprecond = false;
```

Comment:
this should be declared in the main() and passed into the function, and should have a name like use_preconditioned_affine_component.